java/webhook/src/main/java/com/google/hangouts/chat/webhook/App.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2022 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.hangouts.chat.webhook;
+
+// [START hangouts_chat_java_webhook]
```

Comment:
Omit language. Use `hangouts_chat_webhook`.

OK to reuse region tags across languages, and the internal tracking tools are designed for this (helps understand language coverage for a use case.)